docs/tutorials/send_receive_ingress.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: "Exchanging messages over an ssl ingress"  
-description: "Steps to get a producer and a consummer exchanging messages over a deployed broker on kubernetes using an ingress"
+title: "Exchanging messages over an ssl ingress with cert-manager's generated certs"
```

Comment:
The certificates manually generated are still supported. The related documentation should not be removed.